pages/getting_started/create_dashboard.rst

```diff
@@ -1,18 +1,14 @@
 Create Your Dashboard
 ---------------------
 
-You should be at a screen like the one below. If you dozed off or went to cook some meatballs, go to System -> Inputs, select the Syslog UDP input you created, and hit Show messages.
-
-Now it’s go-time.
-
 You’ve got data coming in, let’s add information to a dashboard to better visualize the data we want to see.
 
 Add a Dashboard
 ^^^^^^^^^^^^^^^^^^
 
-We'll start by adding the message count data to a dashboard. Click *Add count to dashboard*, and it will say *No Dashboards, create one?*   Yes!  Click that.
+We will now add a new dashboard, go to *Dashboards* in the top, select *Create dashboard* 
 
-.. image:: /images/gs_11-createdash.png
+.. image:: /images/gs/graylog_dashboard.png
 
 Give your new dashboard a title and description.
```

Comment:
You can't use the ReadTheDocs search or the browser's in-page search on text in a picture.